arch/cpu/arm/cortex-m/trustzone/Makefile.trustzone

```diff
@@ -0,0 +1,4 @@
+# Provide a library of secure functions to the normal world.
+LDFLAGS += -Xlinker --cmse-implib
```

Comment:
If you put the first two lines of LDFLAGS under the `ifeq ($(TRUSTZONE_SECURE_BUILD), 1)`  block in Makefile.cortex-m instead, this Makefile can be removed completely.